modules/kubernetes/pages/k8s-operator/installation.adoc

```diff
@@ -0,0 +1,43 @@
+= Installing TigerGraph Kubernetes Operator
+:description: Instructions on how to install TigerGraph Kubernetes Operator.
+:sectnums:
+
+This page describes the steps to install TigerGraph Kubernetes Operator on your Kubernetes cluster.
+
+The Operator only needs to be installed once for each cluster.
+If you run into the CRD
```

Comment:
Unfinished sentence? Not sure what CRD means here either, maybe explain the acronym